ground/src/main/java/com/google/android/ground/repository/LocationOfInterestRepository.kt

```diff
@@ -73,6 +81,29 @@ constructor(
         Single.error { NotFoundException("Location of interest not found $locationOfInterest") }
       )
 
+  fun createLocationOfInterest(geometry: Geometry, job: Job, surveyId: String): LocationOfInterest {
+    val auditInfo = AuditInfo(authManager.currentUser)
+    return LocationOfInterest(
+      id = uuidGenerator.generateUuid(),
+      surveyId = surveyId,
+      geometry = geometry,
+      job = job,
+      created = auditInfo,
+      lastModified = auditInfo
+    )
+  }
+
+  fun createLocationOfInterestForGeometry(geometry: Geometry, surveyId: String): @Cold Completable =
```

Comment:
Nit: Rename to something that clarified this is applying the mutation, as opposed to creating the object as in `createLocationOfInterest`

Reply:
This has since been removed.